### DISCUSSION TITLE
Home Assistant Configuration

### DISCUSSION THREAD

**Discussion by magichew** [alienatedsec/solis-ha-modbus-cloud#15] [Q&A]
Hi all

I have all of the hardware to accomplish this however I haven't began the home assistant aspect as I have ran out of talent/skill/knowhow to set it up.

I am familiar with getting a HA installation up and running but when it comes to the plugins required and the graphing and stuff to display the information gleaned from the inverter.

Can anyone point me to a tutorial that might help me with setting this up?

**Comment by alienatedsec** [comment]
Try this https://github.com/jevburchell/Solis-Modbus-Auto-Charging-using-Solax by @jevburchell

Should work the same with our hardware too.

**Reply by TikTok90** [reply]
Seems this link is now dead.  Looks like Jev has made all his work private.

**Reply by alienatedsec** [reply]
Not sure why, but the repo has a new owner now - https://github.com/fskelly/Solis-Modbus-Auto-Charging-using-Solax and is maintained by @fskelly

**Reply by fskelly** [reply]
Huh??? Not sure why. Very confused. What did I do? How do I undo this?

**Reply by fskelly** [reply]
I just forked the repo, VERY sorry if I broke something.

**Reply by fskelly** [reply]
Added you @alienatedsec as a colloborator on my fork, again, not sure how I become the owner.

**Reply by alienatedsec** [reply]
@fskelly I don't think you have done anything. His account seems inactive. I will check on Facebook.

**Reply by alienatedsec** [reply]
Hi @fskelly I just realised that I haven't responded in regards to Jev. He had to take a break due to health problems and transferred the ownership straight after.
